Post-install fails on windows because unzip missing

See https://github.com/breach/thrust/issues/227

```
events.js:72
throw er; // Unhandled 'error' event
^
Error: spawn ENOENT
at errnoException (child_process.js:1001:11)
at Process.ChildProcess._handle.onexit (child_process.js:792:34)
npm ERR! node-thrust@0.7.5 postinstall: node scripts/bootstrap.js
npm ERR! Exit status 8
```

--- a/package.json
+++ b/package.json
@@ -9,11 +9,12 @@
   "version": "0.1.5",
   "dependencies": {
     "async": "0.9.x",
+    "fs-extra": "0.12.x",
+    "minimist": "1.1.x",
     "mkdirp": "0.5.x",
     "optimist": "0.6.x",
-    "fs-extra": "0.12.x",
     "request": "2.45.x",
-    "minimist": "1.1.x"
+    "unzip": "^0.1.11"
   },
   "repository": {
     "type": "git",

--- a/scripts/bootstrap.js
+++ b/scripts/bootstrap.js
@@ -18,6 +18,7 @@ var os = require('os');
 var request = require('request');
 
 var common = require('../lib/common.js');
+var unzip = require('unzip');
 
 /******************************************************************************/
 /* CONFIGURATION */
@@ -114,23 +115,9 @@ var install_thrust = function(force, cb_) {
     function(cb_) {
       console.log('Extracting ' + path.join(THRUST_PATH, 
                                             THRUST_RELEASE_FILENAME));
-      var unzip = require('child_process').spawn('unzip', 
-        [ '-oqq', path.join(THRUST_PATH, THRUST_RELEASE_FILENAME) ], {
-          cwd: THRUST_PATH
-        });
-      unzip.stdout.on('data', function (data) {
-        console.log('stdout: ' + data);
-      });
-      unzip.stderr.on('data', function (data) {
-        console.log('stderr: ' + data);
-      });
-      unzip.on('close', function (code) {
-        if(code !== 0) {
-          return cb_(common.err('Extraction failed with code: ' + code,
-                                'boostrap:failed_extraction'));
-        }
-        return cb_();
-      });
+      fs.createReadStream(path.join(THRUST_PATH, THRUST_RELEASE_FILENAME)).pipe(unzip.Extract({
+        path: THRUST_PATH
+      })).on('end', cb_);
     },
     /* Cleaning up */
     function(cb_) {

